src/main/kotlin/ai/digital/integration/server/common/domain/KubeScanner.kt

```diff
@@ -0,0 +1,12 @@
+package ai.digital.integration.server.common.domain
+
+import org.gradle.api.model.ObjectFactory
+import org.gradle.kotlin.dsl.property
+
+@Suppress("UnstableApiUsage")
+open class KubeScanner(objects: ObjectFactory) {
+
+    var enableDebug: Boolean = objects.property<Boolean>().value(false).get()
+
+    var awsRegion: String = objects.property<String>().value("us-east-1").get()
```

Comment:
Please don't define here any default value.
Instead read it from `~/.aws/config`. If it's not defined there and no value explicitly provided, then we will through an error.